src/common/helpers/AuthProvider.ts

```diff
@@ -121,6 +121,6 @@ export default class AuthProvider implements IAuthProvider {
      * @returns {Promise<void>}
      */
     public async logout(): Promise<void> {
-        this.keyHandler.clearStorage()
+        await this.keyHandler.clearStorage()
```

Comment:
Is this right?

Reply:
fixed, now is return `this.keyHandler.clearStorage()` that returns a `Promise<void>`